snuba/clickhouse/native.py

```diff
@@ -81,12 +84,31 @@ def __init__(
         self.client_settings = client_settings
 
         self.pool: queue.LifoQueue[Optional[Client]] = queue.LifoQueue(max_pool_size)
+        self.fallback_pool: queue.LifoQueue[Optional[Client]] = queue.LifoQueue(
+            self.FALLBACK_POOL_SIZE
+        )
         self.__gauge = ThreadSafeGauge(metrics, "connections")
 
         # Fill the queue up so that doing get() on it will block properly
         for _ in range(max_pool_size):
             self.pool.put(None)
 
+        for _ in range(self.FALLBACK_POOL_SIZE):
+            self.fallback_pool.put(None)
+
+    def fallback_pool_enabled(self) -> bool:
+        return (
+            state.get_config("use_fallback_host_in_native_connection_pool", "False")
+            == "True"
```

Comment:
We use 1 and 0 integers for flags. Please do not use strings, so we keep consistency across the code base.